.env.example

```
@@ -0,0 +1,2 @@
+CLIENT_ID=YOUR_CLIENT_ID
```

What do you think about prefixing these env variable names with paypal? 
`PAYPAL_CLIENT_ID` and `PAYPAL_CLIENT_SECRET`. 

I imagine devs could end up with multiple secrets in their env file and this would help distingish that these are specific to paypal.
```suggestion
PAYPAL_CLIENT_ID=YOUR_CLIENT_ID
```